pkg/binlog/filename_test.go

```diff
@@ -64,11 +158,106 @@ func (t *testFilenameSuite) TestVerifyBinlogFilename(c *C) {
 	}
 
 	for _, cs := range cases {
-		err := VerifyBinlogFilename(cs.filename)
+		err := VerifyFilename(cs.filename)
 		if cs.valid {
 			c.Assert(err, IsNil)
 		} else {
 			c.Assert(errors.Cause(err), Equals, ErrInvalidBinlogFilename)
 		}
 	}
 }
+
+func (t *testFilenameSuite) TestGetFilenameIndex(c *C) {
```

Comment:
seems we can combine `TestParseFilename ` and `TestGetFilenameIndex `?

Reply:
done in https://github.com/pingcap/dm/pull/140/commits/a59fbf25459d016f7b969ff085c8076460243619